src/containers/forms/CreateCategoryForm.jsx

```diff
@@ -151,6 +166,18 @@ class CreateCategoryForm extends Component {
         <span className='validate__errors'>
           {this.renderValidationError('value')}
         </span>
+        <div className='formField'>
+          { /* eslint-disable */ }
+          <label className='formField__label--checkbox'>Supports Multiple</label>
+          { /* eslint-disable */ }
+          <input
```

Comment:
@Chris-Maina A select or drop down is not ideal because it is counter-intuitive and not the ideal use case for such a data type. I shall instead try to update the checkbox and make it a toggle with appropriate height